event-sourcing/event-publisher/linked-event-processor/src/main/java/uk/gov/justice/services/eventsourcing/linkedevent/LinkedEventProcessor.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.services.eventsourcing.linkedevent;
+
+import static java.lang.String.format;
+
+import uk.gov.justice.services.eventsourcing.prepublish.LinkedEventFactory;
+import uk.gov.justice.services.eventsourcing.prepublish.MetadataEventNumberUpdater;
+import uk.gov.justice.services.eventsourcing.prepublish.PrePublishRepository;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.Event;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventConverter;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.LinkedEvent;
+import uk.gov.justice.services.messaging.Metadata;
+import uk.gov.justice.subscription.registry.SubscriptionDataSourceProvider;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import javax.inject.Inject;
+
+public class LinkedEventProcessor {
+
+    @Inject
+    private SubscriptionDataSourceProvider subscriptionDataSourceProvider;
```

Comment:
the @Injects should be package protected...and below in same class

Reply:
done